python/lsst/ctrl/mpexec/cli/script/run.py

```diff
@@ -190,4 +190,10 @@ def run(  # type: ignore
 
     f = CmdLineFwk()
     taskFactory = TaskFactory()
+
+    # If we have no output run specified, use the one from the graph rather
+    # than letting a new timestamped run be created.
+    if not args.output_run and qgraphObj.metadata:
+        args.output_run = qgraphObj.metadata["output_run"]
```

Comment:
Maybe check for situation when metadata is present but "output_run" key is not? I guess we need a new ticket to make parts of metadata non-optional.